### PR TITLE
Update scale for BarChart brush

### PR DESCRIPTION
**Pull request by steppy452**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

BarChart using `category` type xAxis is blowing up when `onBrushChange` is called due to scale not having the invert function

Issue Number: N/A

## What is the new behavior?
I basically wrote a scale bend invert function to handle reversing the scaleBend when an invert function does not exist on the scale object.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No - at least I don't think so?
```

## Other information
I'm not 100% sure this doesn't break something else lol

### CODE REVIEW

File: src/common/Brush/Brush.story.tsx [hunk 1]
```diff
@@ -89,7 +90,12 @@ export const Bar = () => {
     <BarChart
       width={450}
       height={45}
-      brush={<ChartBrush disabled={false} />}
+      brush={
+        <ChartBrush
+          disabled={false}
+          onBrushChange={({ domain }) => console.log(domain)}
```
Comment (steppy452) [inline src/common/Brush/Brush.story.tsx:96]: i added this to test the changes

File: src/BarChart/BarChart.tsx [hunk 2]
```diff
@@ -377,7 +377,7 @@ export const BarChart: FC<Partial<BarChartProps>> = ({
               element={brush}
               height={chartHeight}
               width={chartWidth}
-              scale={xScale}
+              scale={yScale}
```
Comment (steppy452) [inline src/BarChart/BarChart.tsx:380]: so, this is what it boiled down to afaict .. but @amcdnl lmk if I'm missing something major here

Comment (amcdnl) [reply src/BarChart/BarChart.tsx:380]: The brush should be on the xscale - so I don't think this is the right fix.

**Review by amcdnl — CHANGES_REQUESTED** [review]

**Comment by steppy452** [comment]
@amcdnl ok - this should work as expected now!

**Review by amcdnl — APPROVED** [review]

**Pull request merged**